server/src/uuid/model/entity/mod.rs

```diff
@@ -802,7 +753,9 @@ impl Entity {
             }) = repository.concrete_uuid
             {
                 if abstract_entity.current_revision_id == Some(revision_id) {
-                    return Err(EntityCheckoutRevisionError::RevisionAlreadyCheckedOut);
+                    return Err(operation::Error::BadRequest {
+                        reason: "revision is already checked out".to_string(),
```

Comment:
We have been lately implementing in a way that such cases returns succesfully early in such cases. What do you think: keep the error or return ok?